test/Microsoft.OpenApi.Tests/Models/OpenApiExampleTests.cs

```diff
@@ -119,7 +120,8 @@ public void SerializeAdvancedExampleAsV3JsonWorks()
           {
             ""href"": ""http://example.com/1"",
             ""rel"": ""sampleRel1"",
-            ""binary"": ""AQID""
+            ""bytes"": ""AQID"",
+            ""binary"": "")*+""
```

Comment:
>)*+ [](start = 26, length = 3)

For the binary case, can we test with a bit more complicated string (something that's not in ASCII)? 

Instead of putting in the byte[] specifically above for the binary case, you can use the UTF8.GetBytes function to get the right byte array.

Reply:
I original use the byte[] { 1, 2, 3 }, but, the output string is not displayable. How can I show that?

Reply:
Yeah, that's because 1, 2, 3 correspond to some control characters in ASCII (thus non-printable).

I was suggesting something along this line:

- Here in line 124, do something like ""Ñ😻😑♮Í☛oƞ♑😲☇éǋžŁ♻😟¥a´Ī♃ƠąøƩ""
- Then, in line 35, do UTF8.GetBytes("Ñ😻😑♮Í☛oƞ♑😲☇éǋžŁ♻😟¥a´Ī♃ƠąøƩ")

---
In reply to: [265777035](https://github.com/Microsoft/OpenAPI.NET/pull/391#discussion_r265777035) [](ancestors = 265777035)

Reply:
:) I changed.